projects/core/src/cart/facade/cart.service.ts

```diff
@@ -1,16 +1,15 @@
 import { Injectable } from '@angular/core';
 
 import { Store, select } from '@ngrx/store';
-
-import { AuthService, Cart, OrderEntry, UserToken } from '@spartacus/core';
-
 import { Observable } from 'rxjs';
 import { filter } from 'rxjs/operators';
 
+import { Cart, OrderEntry } from '../../occ-models/index';
+import { AuthService, UserToken } from '../../auth/index';
+
 import * as fromAction from '../store/actions';
 import * as fromReducer from '../store/reducers';
 import * as fromSelector from '../store/selectors';
-
 import { ANONYMOUS_USERID, CartDataService } from './cart-data.service';
 
 @Injectable()
```

Comment:
For correct typing, Store sohult be `Store<StateWithCart>` instead of Store<CartState>